css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,99 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+  /* Keep the header bar visible when the sidebar is open */
+  height: calc(100vh - var(--layout-header-height-mobile));
+  margin-top: var(--layout-header-height-mobile);
+  padding-bottom: calc(var(--layout-header-height-mobile) / 2);
+
+  display: flex;
```

Comment:
this will be almost always overriden by inline styles on the sidebar- i am not sure  if this would work